book/src/specs/kimchi.md

```diff
@@ -1969,6 +1969,14 @@ pub struct ProofEvaluations<Evals> {
     pub generic_selector: Evals,
     /// evaluation of the poseidon selector polynomial
     pub poseidon_selector: Evals,
+    /// evaluation of the EC addition selector polynomial
+    pub complete_add_selector: Evals,
+    /// evaluation of the EC variable base scalar multiplication selector polynomial
+    pub mul_selector: Evals,
```

Comment:
[minor] EC does not seem to be used anywhere else in the book. I suggest to expand it like so
```suggestion
    /// evaluation of the elliptic curve addition selector polynomial
    pub complete_add_selector: Evals,
    /// evaluation of the elliptic curve variable base scalar multiplication selector polynomial
    pub mul_selector: Evals,
```